README.md

```diff
@@ -121,7 +121,7 @@ installed.
 exiftool is installed using the script at `src/main/script/ExiftoolInstall` and is configured using the properties file
 `tools.properties`.
 
-Tika is installed exclusively through its pom file `tika-pom.xml`. 
+Tika and JHOVE are installed exclusive through their pom files `tika-pom.xml` and `jhove-pom.xml` respectively. 
```

Comment:
`exclusive` -> `exclusively` ?

Reply:
:( That's what happens when you have a commit backlog